### ISSUE TITLE
Behat 3.4 breaks accessing symfony2 parameters in behat.yml

### ISSUE THREAD

**Issue by yakobe** [Behat/Symfony2Extension#125]
After updating to 3.4 it is no longer possible to access symfony2 parameters from the behat.yml as stated in the [docs](http://behat.org/en/latest/cookbooks/integrating_symfony2_with_behat.html#accessing-symfony2-parameters-in-behat-yml)

```
default:
    suites:
        default:
            contexts:
                - FeatureContext:
                    mySymfonyParam: %%my.symfony_param%%
    extensions:
        Behat\Symfony2Extension: ~
```

This now produces an error:
```
[Symfony\Component\DependencyInjection\Exception\ServiceNotFoundException]
  You have requested a non-existent service "my.symfony_param".
```

Also created https://github.com/Behat/Behat/issues/1077 since i am not sure which repo is the cause.

**Comment by bmxlugo** [comment]
@yakobe I have found that commit is related, you need to use env variables 36a9d27#diff-38cb5d6599068c5b98d1464939a93445 or give someone knows another solution.

**Comment by sroze** [comment]
Yes, I can reproduce this problem. It's after this PR: https://github.com/Behat/Behat/pull/1048. Digging into it.

**Comment by sroze** [comment]
Double escaping the argument works tho:
```yaml
default:
    suites:
        default:
            contexts:
                - FeatureContext:
                    mySymfonyParam: %%%%my.symfony_param%%%%
```

**Comment by sroze** [comment]
Fixed in Behat 3.4.3.
